website/docs/r/bot_channel_directline.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Bot"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_bot_channel_directline"
+description: |-
+  Manages an Directline integration for a Bot Channel
+---
+
+# azurerm_bot_connection
+
+Manages a Directline integration for a Bot Channel
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "northeurope"
+}
+
+resource "azurerm_bot_channels_registration" "example" {
+  name                = "example"
+  location            = "global"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  sku                 = "F0"
+  microsoft_app_id    = "${data.azurerm_client_config.current.service_principal_application_id}"
+}
+
+resource "azurerm_bot_channel_directline" "example" {
+  bot_name            = "${azurerm_bot_channels_registration.example.name}"
+  location            = "${azurerm_bot_channels_registration.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+
+  site {
+    site_name  = "default"
+    is_enabled = true
+  }
+}
+```
+
+## Argument Reference
```

Comment:
Could we document the rest of the properties?

Reply:
Yep!